examples/checkout/index.js

```diff
@@ -0,0 +1,70 @@
+$(function() {
+
+  /* Build new ShopifyBuy client
+  ============================================================ */
+  var client = ShopifyBuy.buildClient({
+    apiKey: 'bf081e860bc9dc1ce0654fdfbc20892d',
+    myShopifyDomain: 'embeds',
+    appId: '6'
+  });
+
+  var product;
+  var cart;
+
+  client.createCart().then(function (newCart) {
+    cart = newCart;
+    return client.fetchProduct('3614436099');
+  }).then(function (product) {
+    cart.addVariants({ variant: product.variants[0], quantity: 5 }, { variant: product.variants[1], quantity: 3 }).then(function() {
+      completeUIRendering();
+      bindEventListeners();
+    }).catch(function (errors) {
+      console.log('Fail');
+      console.error(errors);
+    });
+  });
+
+  /* Update UI variables
+  ============================================================ */
+  function completeUIRendering() {
+    $('.cart.empty').toggle(!cart.lineItemCount);
+    $('.cart.not-empty').toggle(cart.lineItemCount !== 0);
+
+    $('.cart.not-empty .sub-total').text(formatAsMoney(cart.subtotal));
+    $('.cart.not-empty .line-item-count').text(cart.lineItemCount);
```

Comment:
`$('.cart.not-empty')` is used repeatedly, maybe set it as as a var then use `.find` if you need to find elements inside it